src/modules/git_commit.rs

```diff
@@ -80,23 +50,27 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
-fn format_tag(symbol: &str, tag_name: &str) -> Option<Result<String, StringFormatterError>> {
-    if tag_name.is_empty() {
-        None
-    } else {
-        Some(Ok(format!("{}{}", &symbol, &tag_name)))
-    }
+fn git_tag(repo: &Repo) -> Option<String> {
+    let git_repo = repo.open();
+    let mut git_head = git_repo.head().ok()?;
+    let head_commit = git_head.peel_to_commit_in_place().ok()?;
```

Comment:
There is a [`head_commit()`](https://github.com/Byron/gitoxide/blob/b362fb594546400e6c42688103df438954df7eeb/git-repository/src/repository/reference.rs#L213) method now.

Generally, `gitoxide` wants to make common tasks simple, and I am happy to add what's seemingly missing, too.

Reply:
Thanks for the heads-up!